packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx

```diff
@@ -157,11 +166,50 @@ export class ChoiceGroupOptionBase extends BaseComponent<IChoiceGroupOptionProps
     );
   }
 
+  private _onLabelWrapperRef(element: HTMLDivElement | null): void {
+    if (element) {
+      const labelWrapperElement = createRef<HTMLDivElement>();
+      labelWrapperElement.current = element;
+      this._labelWrapperElementList.push(labelWrapperElement.current);
+    }
+  }
+
+  private _calculateLabelTextOverflow = (): void => {
+    if (!this._labelWrapperElementList) {
+      return;
+    }
+    for (let labelListIndex = 0; labelListIndex < this._labelWrapperElementList.length; labelListIndex++) {
+      const labelElement = this._labelWrapperElementList[labelListIndex];
+      if (labelElement && hasOverflow(labelElement)) {
+        const node = document.createElement('span');
+        const textNode = document.createTextNode('...');
+        node.appendChild(textNode);
+        this._classNames.labelOverflowWrapper && node.classList.add(this._classNames.labelOverflowWrapper);
+        labelElement.insertAdjacentElement('afterend', node);
```

Comment:
manually manipulating the DOM that react creates is probably asking for trouble. do you need to do this? I don't really udnerstand the logic and there are no comments to explain...

Reply:
I need a way to append an ellipsis to the labelWrapper if text is overflowing. Since I discovered I can't do this with css and still have word/break/wrapping, appending a node when a labelWrapper has overflow is the best I could come up with. 

Now that you have context, if you have another idea for doing this, please share!

Reply:
Another option could be to add this to the component's state and then conditionally render the ellipsis. That would keep all of the DOM manipulations within React.

Reply:
Great idea! This worked, I'll push this change soon.